### ISSUE TITLE
PHPUnit 7 wrong dependencies 

### ISSUE THREAD

**Issue by richardhughes** [povils/phpmnd#64]
Get the following message when trying to install PHPUnit 7 with this package. 

``` 
Problem 1
    - Installation request for povils/phpmnd ^1.1 -> satisfiable by povils/phpmnd[1.1.x-dev, v1.1.0, v1.1.1].
    - Conclusion: don't install phpunit/php-timer 2.0.0
    - phpunit/phpunit 7.0.0 requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - phpunit/phpunit 7.1.x-dev requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - phpunit/phpunit 7.0.x-dev requires phpunit/php-timer ^2.0 -> satisfiable by phpunit/php-timer[2.0.0, 2.0.x-dev].
    - Conclusion: don't install phpunit/php-timer 2.0.x-dev
    - Installation request for phpunit/phpunit ^7 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.x-dev, 7.1.x-dev].
```

Could we update the dependency to php-timer ^2.0? I'll submit a PR for this later tonight

**Comment by juliangut** [comment]
Yep, facing that problem just now

dependency should be "phpunit/php-timer": "^1.0|^2.0"

**Comment by povils** [comment]
Solved with release 2.0.0
